overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -1704,7 +1704,7 @@
         "0:10": {
           "autoConsume:1": 0,
           "consume:1": 0,
-          "entryLogic:8": "AND",
+          "entryLogic:8": "OR",
```

Comment:
This isn't clear, as we don't use Entry Logic elsewhere. Split this into two tasks instead, make the Task Logic OR, and then, optional, add a note in the quest description itself (at the beginning of the quest description, in italics usually).